test5.js

```diff
@@ -0,0 +1,12 @@
+var foo = '' ;
+
+if(foo  === "bar") {
```

Comment:
Codacy has a fix for the issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26923)
```suggestion
if(foo === "bar") {
```